flink-yarn/src/main/java/org/apache/flink/yarn/cli/FlinkYarnSessionCli.java

```diff
@@ -108,6 +111,11 @@
 
 	private final Options ALL_OPTIONS;
 
+	private static final String fileName = "yarn-app.ini";
+	private static final String cookieKey = "secureCookie";
```

Comment:
Please see Yarn properties which should be used for this purpose: `writeYarnProperties(..)`.

Please no new configuration file.